cypress/fixtures/devices.json

```diff
@@ -0,0 +1,167 @@
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867900000",
+               "downlink_frequency":"867900000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            }
+         ],
+         "adr_ack_limit_exponent":"ADR_ACK_LIMIT_64",
+         "adr_ack_delay_exponent":"ADR_ACK_DELAY_32",
+         "ping_slot_data_rate_index_value":3
+      },
+      "desired_parameters":{
+         "max_eirp":16,
+         "adr_nb_trans":1,
+         "rx1_delay":1,
+         "rx2_data_rate_index":3,
+         "rx2_frequency":"869525000",
+         "ping_slot_frequency":"869525000",
+         "channels":[
+            {
+               "uplink_frequency":"868100000",
+               "downlink_frequency":"868100000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"868300000",
+               "downlink_frequency":"868300000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"868500000",
+               "downlink_frequency":"868500000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867100000",
+               "downlink_frequency":"867100000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867300000",
+               "downlink_frequency":"867300000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867500000",
+               "downlink_frequency":"867500000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867700000",
+               "downlink_frequency":"867700000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            },
+            {
+               "uplink_frequency":"867900000",
+               "downlink_frequency":"867900000",
+               "max_data_rate_index":5,
+               "enable_uplink":true
+            }
+         ],
+         "adr_ack_limit_exponent":"ADR_ACK_LIMIT_64",
+         "adr_ack_delay_exponent":"ADR_ACK_DELAY_32",
+         "ping_slot_data_rate_index_value":3
+      },
+      "lorawan_version":"MAC_V1_0_2"
+   },
+   "session":{
+      "dev_addr":"26012F9D",
+      "keys":{
+         "session_key_id":"eVNOD7hXFrwVAnqypet9Og==",
+         "f_nwk_s_int_key":{
+            "key":"E276C2B1A158B29C85996EEEDDDEC551"
+         },
+         "app_s_key":{
+            "key":"E82B8166DFB4DA501E169D6380792512"
+         }
+      },
+      "last_f_cnt_up":64,
+      "last_n_f_cnt_down":1,
+      "started_at":"2021-05-21T11:38:08.078114300Z"
+   }
+}
```

Comment:
For a realistic test, you should use a file that has more than just one end device. I would say it should be at least three entries.